data-protocols/ion/ion-core/build.gradle.kts

```diff
@@ -0,0 +1,24 @@
+plugins {
+    `java-library`
+}
+
+val jwtVersion: String by project
+val okHttpVersion: String by project
+
+repositories {
+    mavenCentral()
+}
+
+dependencies {
+
+    implementation("com.squareup.okhttp3:okhttp:${okHttpVersion}")
+    implementation("com.auth0:java-jwt:${jwtVersion}")
+    implementation("com.github.multiformats:java-multihash:1.2.0")
+
+    api("com.nimbusds:nimbus-jose-jwt:8.20.1")
```

Comment:
At this time, the RegistrationService (`extensions/iam/registration-service`) has a hard dependency on the `ECKey` class from the Nimbus library, because that's what we're using for the ION implementation, in particular for the `DidDocument.java`.